cmd/root.go

```diff
@@ -48,6 +51,28 @@ func setupLogger() *zap.Logger {
 	}
 	return logger
 }
+func modifyToSentryLogger(log *zap.Logger, client *sentry.Client) *zap.Logger {
+	cfg := zapsentry.Configuration{
+		Level: zapcore.ErrorLevel, //when to send message to sentry
```

Comment:
Yes, not only in debug mode, even without debug mode it will send you a notification in case of error.

Reply:
But there are many errors which we throw like user app errors or others. I don't think it is good to send a notification when we also know the cause of the problem.

Reply:
Error logs will be sent even when there is no panic, since they are attached to the logger now. Regarding the first part of your question, will discuss tomorrow.

Reply:
Please mention the requirements once discussed.